### ISSUE TITLE
Moving Cameras w/ people in and out of frame

### ISSUE THREAD

**Issue by jacq819** [kijai/ComfyUI-segment-anything-2#33]
Hi! I have a video where people are coming in and out of frame and I want to generate a mask for all persons found in the clip. My prompt in the workflow using Florence2 is "person". It appears that the moment the first person found exits the frame, the segmentation stops for the remainder of the clip, even though more people appear eventually. I can see that Florence2 can detect people beyond that point for the duration of the clip, but the segmentation drops off and rather than saving out a mask clip that is the same frame length as my source video, its much shorter. I've tried "people" and "humans" for alt prompts but same result. Any suggestions?

**Comment by kijai** [comment]
Yeah the video segmenter doesn't support that yet, it would need adding new points in the moment the subject re-appears. Alternative for now could be using the single_image segmenter in batch mode, that does each frame individually, but that requires pretty much perfect tracking.
